### DISCUSSION TITLE
GraphQL API is missing introspection types

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#102] [Help] [answered]
<img width="1088" alt="image" src="https://user-images.githubusercontent.com/193115/213925037-45ade76b-0d84-4152-be16-373b5d8ec57a.png">

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi, @haf! I was able to fetch introspection types @ `haf-se.swell.store/graphql` with a POST to `IntrospectionQuery`. You can see an example of that request by checking your Network console tab when accessing your [GraphQL Playground ](haf-se.swell.store/playground).

If you have added/updated models with custom fields, you can rebuild the GraphQL schema by using `Cache-control: no-cache` in the request headers.
